content/blog/2020-08-28-api.Rmd

```diff
@@ -401,6 +402,9 @@ this is the `fb-survey` data source's `smoothed_hh_cmnty_cli` signal.
 import covidcast
 from datetime import date
 import matplotlib.pyplot as plt
+import os
+
+covidcast.use_api_key(os.environ['API_KEY']) # for more on API keys, see: https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html
```

Comment:
So I thought this plot is not showing because [the plot_choropleth method used in this chunk ](https://github.com/cmu-delphi/covidcast/blob/d2aa655c01e8cc411202fad82e89f6ad9f091d0d/Python-packages/covidcast-py/covidcast/plotting.py#L104) has been deprecated and has been generalized to plot(). 
But it seems like it's something with the env as I upgrade covidcast to latest version so API keys can be used. I'm still digging but I think I'm getting pretty close to the problem.

Reply:
workaround at [9bf703d](https://github.com/cmu-delphi/www-main/pull/891/commits/9bf703d50d2806d594b205f526a39d24b1cd0237)